components/director/pkg/graphql/schema.graphql

```diff
@@ -1,552 +1,532 @@
-# Directives
+directive @hasScopes(path: String!) on FIELD_DEFINITION
```

Comment:
Please add comment that this file is modified automatically with `hasScopes` directives